ide/static/js/signup.js

```diff
@@ -0,0 +1,298 @@
+import React from "react";
+import DjangoCSRFToken from "django-react-csrftoken";
+import $ from "jquery";
+
+class Signup extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      error_username: 0,
+      error_email: 0,
+      error_password1: 0,
```

Comment:
@sayamkanwar please rename this variables, try not to use 1 or 2 with variable for multiple variables for same purpose

Reply:
Actually I was trying to maintain consistency as these variables were used in `/accounts/signup/`. So should I still change these?
<img width="1430" alt="screenshot 2018-12-11 at 11 01 11 pm" src="https://user-images.githubusercontent.com/10847009/49818840-cc0c9380-fd99-11e8-83c0-1b097a7bcc31.png">

Reply:
Yes

Reply:
Okay